game/operation/operation.py

```diff
@@ -503,6 +505,26 @@ def generate_lua(
                         },
                     }
 
+        for cp in cls.game.theater.controlpoints:
+            for ground_object in cp.ground_objects:
+                if ground_object.might_have_aa and not ground_object.is_dead:
+                    for g in ground_object.groups:
+                        threat_range = ground_object.threat_range(g)
+
+                        if not threat_range:
```

Comment:
Does this work they way you have it now? Do you not need to define `__bool__` for `Distance`? aiui this will always be `True`.

If I'm wrong this is ready to merge, but it looks like this branch will never be hit.

Reply:
I checked the Distance class and it seems there is already a `__bool__` that perfectly fits. https://github.com/Khopa/dcs_liberation/blob/923459c88bbf296c0cc969a0ee2969f2af594fbb/game/utils.py#L76

My intention was to filter out units that have a threat range of 0. Please correct me if the code does not what I intend. I`m a python rookie.

Reply:
I don't know how I missed that when I went to look for it :(